integration-test/background/atb.js

```diff
@@ -178,7 +48,7 @@ describe('search workflow', () => {
     let twoWeeksAgoAtb
 
     beforeAll(async () => {
```

Comment:
How come we're not moving all the atb tests over?

Reply:
I didn't want to spend too much time adding those for this PoC. Looking at those remaining tests, I think some additional thought would be needed for how to move them to an offline test (and if that makes sense), as they seem to depend on the SERP's behavior. I'm not familiar enough with the ATB dynamics to do that myself.

Reply:
Ah OK, it's just I figure these questions are going to come up the first time someone needs to write a new integration test. If we're going to merge this, I'd like to figure out how we expect folks to write tests going forward.

Do the new Playwrite tests have to be 100% offline? I'm wondering if we could simply move the remaining atb tests over into the same file (maybe with a TODO comment), even though they require internet access.